jplag.cli/src/main/java/de/jplag/CLI.java

```diff
@@ -169,9 +192,24 @@ public JPlagOptions buildOptionsFromArguments(Namespace namespace) {
             clusteringBuilder.preprocessor(Preprocessing.THRESHOLD);
             clusteringBuilder.preprocessorPercentile(threshold);
         });
-        options.setClusteringOptions(clusteringBuilder.build());
 
-        return options;
+        return new JPlagOptions( //
+                language, //
+                comparisonMode.orElseThrow(), //
```

Comment:
If we use `orElse` when constructing the comparisonMode, we can obtain directly an unwrapped value and thus not need this anymore.